.github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml

```diff
@@ -0,0 +1,106 @@
+  VERSION_MAJOR: "6"
+  VERSION_MINOR: "0"
+  CURRENT_BRANCH: ${{ GITHUB.HEAD_REF }}
+  ODS_CURRENT_BRANCH: ${{ github.event.client_payload.ref }}
+  EDFI_SUITE_NUMBER: "Suite3"
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - name: Setup .NET
+      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
+      with:
+        dotnet-version: 6.0.x
+    - name: Support longpaths
+      run: git config --system core.longpaths true
+    - name: Checkout Ed-Fi-ODS-Implementation
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
+          path: Ed-Fi-ODS-Implementation/
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+            $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+            Write-Host "Pattern Name is " $patternName -fore GREEN
+            $is_pull_request_branch = 'False'
+            $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
+            if ($is_pull_request_branch -eq $true) {
+              git fetch origin ${{ env.CURRENT_BRANCH }}
+              git checkout ${{ env.CURRENT_BRANCH }}
+            }
+    - name: Checkout Ed-Fi-ODS
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
+          path: Ed-Fi-ODS/
+    - name: Is pull request branch exists in Ed-Fi-ODS
+      working-directory: ./Ed-Fi-ODS/
+      shell: pwsh
+      run: |
+           $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+           Write-Host "Pattern Name is " $patternName -fore GREEN
+           $is_pull_request_branch = 'False'
+           $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
+           Write-Host "pull request branch is '$($is_pull_request_branch -eq $true)'" -fore GREEN
+           $odsBranch = '${{ env.ODS_CURRENT_BRANCH }}'
+           if($odsBranch -ne ''){
+              git fetch origin $odsBranch
+              git checkout $odsBranch
+           } else {
+              if ($is_pull_request_branch -eq $true) {
+                Write-Host "pull request branch " + ${{ env.CURRENT_BRANCH }} + "exists" -fore GREEN
+                git fetch origin ${{ env.CURRENT_BRANCH }}
+                git checkout ${{ env.CURRENT_BRANCH }}
+              }
+           }
+    - name: Start PostgreSQL
+      run: |
+        $pgService = Get-Service -Name postgresql*
+        Set-Service -InputObject $pgService -Status running -StartupType automatic
+        Start-Process -FilePath "$env:PGBIN\pg_isready" -Wait -PassThru
+    - name: ODS/API InitDev, SdkGen, Integration Test, Unit Test, Package
```

Comment:
Can you remove package at the end of name: ODS/API InitDev, SdkGen, Integration Test, Unit Test, Package

Reply:
This has been removed.